js-app/types/FactoryContract.ts

```diff
@@ -90,6 +91,7 @@ export interface WalletInfo {
     code_id: number;
     guardians: Addr[];
     is_frozen: boolean;
+    label: string;
```

Comment:
Since now `label` is a required param, you also need to give a value for it inside tests, otherwise they will error out